src/object-inspector/ObjectInspector.js

```diff
@@ -32,7 +34,8 @@ const createIterator = (showNonenumerable, sortObjectKeys) => {
       }
     } else {
       const keys = Object.getOwnPropertyNames(data);
-      if (sortObjectKeys === true) {
+      if (sortObjectKeys === true && !isArray) {
```

Comment:
would you mind updating README.md too? adding comment that `sortObjectKeys(true)` sort keys by alphabetical order except for arrays